trigger/host.go

```diff
@@ -194,24 +218,48 @@ func (t *hostTemplateData) hostSlackPayload(messageString string, linkTitle stri
 }
 
 func (t *hostTriggers) hostExpiration(sub *event.Subscription) (*notification.Notification, error) {
-	if t.host.NoExpiration {
-		return nil, nil
-	}
-	timeZone := time.Local
-	if sub.OwnerType == event.OwnerTypePerson {
-		userTimeZone, err := getUserTimeZone(sub.Owner)
-		if err != nil {
+	switch t.event.EventType {
+	case event.EventHostExpirationWarningSent:
+		if t.host.NoExpiration {
```

Comment:
I'd recommend breaking these into two different helper functions, for clarity / readability, even if you'd like to reuse the same host trigger (which seems fine)

Reply:
Done, and also deduplicated the copy-pasted time zone logic.